### ISSUE TITLE
Top-1 and Top-5 Accuracy

### ISSUE THREAD

**Issue by Zzlongjuanfeng** [Res2Net/Res2Net-PretrainedModels#20]
Hi, 
Thanks for releasing pretrained models. I am trying to reproduce the result of Res2Net (top-1 and top-5 accuracy). I am using the official PyTorch training code.
1. Is the reported result evaluated on the validation set? or test set?
2. If the result is evaluated on the test set, where could I submit my results?
Thank you!

**Comment by gasvn** [comment]
The model is evaluated on the valiedation set.  And we will release a improved version of Res2Net within the next few weeks. It has more than 2% improvement on ImageNet top-1 acc. (Without using se block and other extra blocks.)

**Comment by Zzlongjuanfeng** [comment]
Thanks for your quick reply. 
Looking forward to that.
